Makefile

```diff
@@ -142,6 +151,8 @@ QEMU_OPTS_COMMON= -smbios type=1,serial=31415926 -m $(QEMU_MEMORY) -smp 4 -displ
         -netdev user,id=eth1,net=$(QEMU_OPTS_NET2),dhcpstart=$(QEMU_OPTS_NET2_FIRST_IP) -device virtio-net-pci,netdev=eth1
 QEMU_OPTS_CONF_PART=$(shell [ -d "$(CONF_PART)" ] && echo '-drive file=fat:rw:$(CONF_PART),format=raw')
 QEMU_OPTS=$(QEMU_OPTS_COMMON) $(QEMU_OPTS_$(ZARCH)) $(QEMU_OPTS_CONF_PART)
+# -device virtio-blk-device,drive=image -drive if=none,id=image,file=X
+# -device virtio-net-device,netdev=user0 -netdev user,id=user0,hostfwd=tcp::1234-:22
```

Comment:
Is this some leftover from some experimentation?

Reply:
Yes, but I wanted to keep it around as a reminder for (future self among others) as to one of the options this could be if an image is not cooperating.